crates/weaver_resolved_schema/src/error.rs

```diff
@@ -20,6 +22,13 @@ pub enum Error {
     /// A generic container for multiple errors.
     #[error("Errors:\n{0:#?}")]
     CompoundError(Vec<Error>),
+
+    /// A generic error identifying a feature that has not yet been implemented.
+    #[error("Not Implemented: {message}")]
+    NotImplemented {
```

Comment:
is this  still used?

Reply:
Will check

Reply:
Based on keeping the `BodySpec::Value { value: _ }` then yes, as there are some event producers (and SDKs) that have been talking about just sending a string.

This is also part of where / how the body definition (might) be extended as per the [discussion here](https://github.com/open-telemetry/semantic-conventions/issues/755)